[0.000000] (-) TimerEvent: {}
[0.000678] (test_pkg) JobQueued: {'identifier': 'test_pkg', 'dependencies': OrderedDict()}
[0.000760] (test_pkg) JobStarted: {'identifier': 'test_pkg'}
[0.098391] (-) TimerEvent: {}
[0.187147] (test_pkg) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 102, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 94, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 192, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 129, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'-W\', \'ignore:setup.py install is deprecated\', \'setup.py\', \'--help-commands\'] to pass: Traceback (most recent call last):\n  File "/home/marsrover/mars_ws/test_pkg/setup.py", line 1, in <module>\n    from setuptools import find_packages, setup\n  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 16, in <module>\n    import setuptools.version\n  File "/usr/lib/python3/dist-packages/setuptools/version.py", line 1, in <module>\n    import pkg_resources\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3267, in <module>\n    def _initialize_master_working_set():\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3241, in _call_aside\n    f(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3279, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 564, in _build_master\n    ws = cls()\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 557, in __init__\n    self.add_entry(entry)\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 613, in add_entry\n    for dist in find_distributions(entry, True):\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2077, in find_on_path\n    for dist in factory(fullpath):\n  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2135, in distributions_from_metadata\n    if len(os.listdir(path)) == 0:\nPermissionError: [Errno 13] Permission denied: \'/usr/local/lib/python3.10/dist-packages/certifi-2024.7.4.dist-info\'\n\n'}
[0.187974] (test_pkg) JobEnded: {'identifier': 'test_pkg', 'rc': 1}
[0.198041] (-) EventReactorShutdown: {}
